doc/examples/scripts/structure/leontis_westhof.py

```diff
@@ -0,0 +1,89 @@
+"""
+Leontis-Westhof Nomenclature
+============================
+
+In this example we plot a secondary structure diagram annotated with 
+Leontis-Westhof nomenclature of the sarcin-ricin loop from E. coli (PDB
```

Comment:
I think a citation of the corresponding Leontis-Westhof article would be helpful here.